packages/cli/src/cmds/init/initUtils.ts

```diff
@@ -290,10 +299,9 @@ export async function buildSnapManifest(
       );
     }
 
-    return [
-      manifest,
-      { dist, outfileName, src: packageJson.main || 'src/index.js' },
-    ];
+    const src = packageJson.main || 'src/index.js';
```

Comment:
This can be reverted now to the way it was before.

Reply:
Yes, reverted it now.